docs/ref/models/fields.txt

```diff
@@ -1721,6 +1721,13 @@ Field API reference
       value)
     * when it saves to the database (Python value -> database backend value)
 
+    .. versionadded:: 1.10
+
+    .. method:: rel_db_type(connection)
+
+        Returns the database column data type for the ForeignKey / OneToOneField,
```

Comment:
Add backticks around field names.
Is this method relevant for custom fields? As it stands now, it's a bit unclear why any reader would care about this method. What does "database column data type" mean? Could you given an example. I think this is the type of information I'd expect in the custom-model-fields howto doc.
Probably should be mentioned in the 1.10 release notes too.